pkgs/by-name/gn/gnum4/package.nix

```
@@ -11,15 +11,13 @@
 
 stdenv.mkDerivation rec {
```

```suggestion
stdenv.mkDerivation (finalAttrs: {
```
Awkward to tell this to a Nix veteran, but let's modernize 😅